### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by carmen-lee**
# [Update README.md for design doc 2]
> Team member: @carmen-lee 

## Description
Reiterated abstract and reformatted to be more eye-catching rather than a block paragraph

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# VolunteerOne
+
+VolunteerOne is a peer-based mobile app that aims to connect volunteers to their communities through posted opportunities and events. Its purpose is to **increase community engagement**, **emphasize input** from volunteers on their skills, interests, and experience to match suitable opportunities and **create more impactful contributions**. 
+
+## Features 
+* Interactive opportunity finder
+* Clear communication channels between organizations and volunteers
+* Hour tracking for groups looking to volunteer together
+* Experience sharing with friends and friends
```
Comment (Rahgid) [inline README.md:9]: nit. Is this supposed to be friends and something else?

**Review by Rahgid — APPROVED** [review]
Just have a minor comment on the last line, still going to approve.
I think it'd be a good idea to add some screenshots here as well, but maybe that can be after we having a working demo.

**Review by lazeroffmichael — APPROVED** [review]
Would it be possible to get the logo on here as well :)

**Review by cerasamson — APPROVED** [review]
Added logo but other than that all looks good

**Pull request merged**